utilities.tex

```diff
@@ -276,28 +276,45 @@
 
   @\newtxt{// \ref{comparisons}, comparisons:}@
   template <class T = void>
-    @\added{requires EqualityComparable<T>() || Same<T, void>()}@
-  struct equal_to;
+    @\oldtxt{requires EqualityComparable<T>() || Same<T, void>()}@
+  struct equal_to; @\newtxt{// not defined}@
+  @\newtxt{template<EqualityComparable T> struct equal_to<T>;}@
 
   template <class T = void>
-    @\added{requires EqualityComparable<T>() || Same<T, void>()}@
-  struct not_equal_to;
+    @\oldtxt{requires EqualityComparable<T>() || Same<T, void>()}@
+  struct not_equal_to; @\newtxt{// not defined}@
+  @\newtxt{template<EqualityComparable T> struct not_equal_to<T>;}@
 
   template <class T = void>
-    @\added{requires \added{Strict}TotallyOrdered<T>() || Same<T, void>()}@
-  struct greater;
+    @\oldtxt{requires StrictTotallyOrdered<T>() || Same<T, void>()}@
+  struct greater; @\newtxt{// not defined}@
+  @\newtxt{template<StrictTotallyOrdered T> struct greater<T>;}@
+  @\newtxt{template<class T> struct greater<T*>;}@
 
   template <class T = void>
-    @\added{requires \added{Strict}TotallyOrdered<T>() || Same<T, void>()}@
-  struct less;
+    @\oldtxt{requires StrictTotallyOrdered<T>() || Same<T, void>()}@
+  struct less; @\newtxt{// not defined}@
+  @\newtxt{template<StrictTotallyOrdered T> struct less<T>;}@
+  @\newtxt{template<class T> struct less<T*>;}@
 
   template <class T = void>
-    @\added{requires \added{Strict}TotallyOrdered<T>() || Same<T, void>()}@
-  struct greater_equal;
+    @\oldtxt{requires StrictTotallyOrdered<T>() || Same<T, void>()}@
+  struct greater_equal; @\newtxt{// not defined}@
+  @\newtxt{template<StrictTotallyOrdered T> struct greater_equal<T>;}@
+  @\newtxt{template<class T> struct greater_equal<T*>;}@
 
   template <class T = void>
-    @\added{requires \added{Strict}TotallyOrdered<T>() || Same<T, void>()}@
-  struct less_equal;
+    @\oldtxt{requires StrictTotallyOrdered<T>() || Same<T, void>()}@
+  struct less_equal; @\newtxt{// not defined}@
+  @\newtxt{template<StrictTotallyOrdered T> struct less_equal<T>;}@
+  @\newtxt{template<class T> struct less_equal<T*>;}@
+
+  @\newtxt{template<> struct equal_to<void>;}@
+  @\newtxt{template<> struct not_equal_to<void>;}@
+  @\newtxt{template<> struct greater<void>;}@
+  @\newtxt{template<> struct less<void>;}@
+  @\newtxt{template<> struct greater_equal<void>;}@
+  @\newtxt{template<> struct less_equal<void>;}@
```

Comment:
I guess the obvious answer is "because the detailed descriptions are organized that way, and the descriptions are that way because that's what the Standard does," but I've always found that fact odd as well.